File: vue-vben-admin/apps/web-naive/src/api/entity/SysUser.ts
export interface SysUser {
  orgId: string  | null | undefined ;    //  组织机构Id
  userCode: string  | null | undefined ;    //  账号
  password: string  | null | undefined ;    //  密码
  salt: string  | null | undefined ;    //  密码加密因子
  userName: string  | null | undefined ;    //  名称
  avatar: string  | null | undefined ;    //  头像
  phone: string  | null | undefined ;    //  联系电话
  email: string  | null | undefined ;    //  电子邮箱
  isAdmin: number  | null | undefined ;    //  是否是超级管理员
  isEnable: number  | null | undefined ;    //  是否启用
  lastLoginTime: string  | null | undefined ;    //  上次登录时间
  lastIpAdress: string  | null | undefined ;    //  上次登录IP
  tenantId: string  | null | undefined ;    //  租户GUID
  id: string  | null | undefined ;    //  主键
  createdId: string  | null | undefined ;    //  创建人GUID
  createdTime: string  | null | undefined ;    //  创建时间
  creator: string  | null | undefined ;    //  创建人
  updatedTime: string  | null | undefined ;    //  更新时间
  updatedId: string  | null | undefined ;    //  更新人GUID
  updator: string  | null | undefined ;    //  更新人
}

File: src/SimpleApp/Simple.AdminApplication/UserMng/Entities/SysUser.cs
namespace Simple.AdminApplication.UserMng.Entities
{
    public class SysUser : DefaultTenantEntity
    {
        /// <summary>组织机构Id</summary>
        public Guid OrgId { get; set; }

        /// <summary>账号</summary>
        public string UserCode { get; set; }

        /// <summary>密码</summary>
        public string Password { get; set; }

        /// <summary>密码加密因子</summary>
        public string Salt { get; set; } = string.Empty;

        /// <summary>名称</summary>
        public string UserName { get; set; } = string.Empty;

        /// <summary>头像</summary>
        public string Avatar { get; set; }

        /// <summary>联系电话</summary>
        public string Phone { get; set; }

        /// <summary>电子邮箱</summary>
        public string Email { get; set; }

        /// <summary>是否是超级管理员</summary>
        public int IsAdmin { get; set; } = 0;

        /// <summary>
        /// 是否启用
        /// </summary>
        public int IsEnable { get; set; }

        /// <summary>
        /// 上次登录时间
        /// </summary>
        public DateTime? LastLoginTime { get; set; }

        /// <summary>
        /// 上次登录IP
        /// </summary>
        public string LastIpAdress { get; set; }
    }
}